admin/src/components/product-add-item-propetries/Product-add-item-propetries.js

```diff
@@ -37,7 +37,12 @@ const AddProductPropetries = ({ setSizeModel, setProductModel, sizeModel, produc
         });
     };
 
-    const propetryTextFields = Object.keys(sizeModel).map(name => (
+    const setSKU = (product, size) => {
+        sizeModel.sku = `${product.catalog[0]}-${product.title.padEnd(5, '0').slice(0,5)}-${product.category.slice(0, 2)}-${product.color.padEnd(5,'0').slice(0,5)}-${size.size.padStart(2,'0')}`.toUpperCase()
```

Comment:
add comment + separate by row or move to function formatSKU = (product, size) => {}